lib/examples/CMakeFiles/CMakeDirectoryInformation.cmake

```diff
@@ -0,0 +1,16 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
also here you must have done an "in-source" build, please remove all file in the `lib/examples/CMakeFiles` folder from the repository.

Reply:
Removed.